tests/data_quality/test_unseen.py

```diff
@@ -231,6 +231,23 @@ def test_results_size_of_downpayment_values(unseen_value_result):
     ]
 
 
-def test_results_size_of_downpayment_alerts(unseen_value_result):
+def test_results_size_of_downpayment_alerts(unseen_value_result):  # noqa: D103
     res = unseen_value_result.filter(column_names='size_of_downpayment').to_df()
     assert list(res[('size_of_downpayment', 'alert')]) == [False] * 10 + [True] * 10
+
+
+def test_oo_behavior_unseen_value_calculator_calculate():  # noqa: D103
```

Comment:
Name could be a bit more descriptive, as this is a weird thing to test.

Maybe something like test_input_dataframes_are_not_altered_by_calculator